### PR TITLE
Update "Display the uncompressed size of a resource" in "Network features reference"

### PR DESCRIPTION
**Pull request by pankajparashar**
Rendered article section for review:
Public approx Markdown preview:
https://github.com/pankajparashar/edge-developer/blob/patch-6/microsoft-edge/devtools-guide-chromium/network/reference.md#display-the-uncompressed-size-of-a-resource
Internal fully rendered:
https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/network/reference?branch=pr-en-us-2940#display-the-uncompressed-size-of-a-resource

This long webpage might not scroll to correct section, so you might need to Find the heading:
**Display the uncompressed size of a resource**

Before:
https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/network/reference#display-the-uncompressed-size-of-a-resource

AB#47822717

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/network/reference.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/network/reference.md:602]: @captainbrosset - KB or kB?

https://en.wikipedia.org/wiki/Kilobyte
https://en.wikipedia.org/wiki/Binary_prefix

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/network/reference.md:602]: keeping kB per png

File: microsoft-edge/devtools-guide-chromium/network/reference.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/network/reference.md:1]: this PR stops importing ...compare.png and adds new file ...size.png.  ok?  
in this PR's branch, is the ...compare.png used? __  if not, delete

File: microsoft-edge/devtools-guide-chromium/network/reference.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/network/reference.md:600]:
```suggestion
Select the **Big request rows** checkbox, and then examine the bottom value of the **Size** column.
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/check-checkbox-check-mark

**Review by mikehoffms — APPROVED** [review]
Writer/Editor review, after pushed commits

**Comment by captainbrosset** [comment]
Thank you @pankajparashar for the continued improvements in the devtools docs. They're very appreciated.

However, for this particular PR, I would prefer if we held off for two weeks. We have a huge PR coming up that we hope to merge in 2 weeks from now. This PR changes practically all images in the devtools docs, and a lot of the text instructions too, in order to make our docs match the new Focus Mode UI of devtools.

The particular article and image you are changing in this PR has been modified in the Focus Mode PR, which means that if we merge this one, it'll be a complex conflict to resolve in the Focus Mode PR, and we'll have to redo the screenshot once more.

So I see two possibilities:

* Either we hold on this PR, not merge it for another two weeks, and then (after the Focus Mode PR has landed), we pick it up and merge it then.
* Or you make this PR target the Focus Mode branch instead of `main`, and change your screenshot to show the Focus Mode UI instead of (what we now call) the legacy UI. The Focus Mode PR is: https://github.com/MicrosoftDocs/edge-developer/pull/2724

Let me know what you think.
Thanks again!

**Comment by pankajparashar** [comment]
Absolutely makes sense. I have a few more PRs on the way but now I think its better to hold off and wait for the Focus mode docs to be live first, before merging this PR.

I'll go with option #1. Thanks & looking forward for the new docs!

**Comment by captainbrosset** [comment]
Thanks for your understanding. Marking as on-hold.

**Pull request closed without merging**
